antarest/study/storage/rawstudy/model/filesystem/config/thermal.py

```diff
@@ -135,15 +135,15 @@ def get_id(self) -> str:
     )
     min_up_time: int = Field(
         default=1,
-        ge=1,
```

Comment:
Note that to make it have the same behaviour as the simulator, I think we should also crop it to 1 or 168 if it's outside of the range [1,168].